tests/jballerina-unit-test/src/test/resources/test-src/workers/wait-actions-negative.bal

```diff
@@ -90,6 +90,59 @@ function waitForAllTest() {
     restRec2 result16 = wait {id: f1, name: f2, age: f4};
 }
 
+type WaitResult record {|
+    boolean|error producer;
+    boolean consumer;
+|};
+
+function waitForAllTest2() {
+    worker Producer1 returns boolean|error {
+        int i = 100;
+        i -> Consumer;
+        error? unionResult = flush Consumer1;
```

Comment:
Do we really need all of this logic? Not required for the test, right? Shall we simplify?